Updated officially supported tournaments rules/information article (#4941)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.

### What is considered 'staff' as far as tournament management goes?

We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.

Commentators are not considered by us as staff, and may play or interact in tournaments freely so long as they are not directly involved any managerial aspect of running the tournament or violate anything mentioned above.

We have historically had issues where unscrupulous organizers have geared tournaments specifically (ie: picked maps they're familiar with, adjusted seeds/brackets, etc) to farm profile badges for them and their friends. While we understand that most groups will not do this, we must enforce this restriction to keep things fair for everyone.

> ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems?

Reviewer: shouldn't there be a clause here about mappool showcase replays/playtesting? this has been a question i've been asked several times by newer hosts where i always concluded with "OWC does it, so it should be fine". it's a bit confusing as mappooling is something usually only managed by the higher ups in staff but since refereeing after being eliminated is not allowed, having an influence on the mappool also shouldn't but as mentioned before OWC (and possibly other world cups which i haven't checked) get eliminated players to help

Reviewer: oops just noticed i commented on more lines than needed, this was meant for only line 136 (im github noob sorry)

Reviewer 2: I personally don't understand the rule that eliminated players cannot join staff afterwards. Sounds over-cautious.

Reviewer 3: I would argue that joining the staff post-elimination looks bad in general but I guess we can have a debate about this. I personally have no issues in regards to having them being play-testers or replay makers.

To join as map selectors or refs I don't like the idea as much because, as former players, they could affect the outcome of the tournament (by picking stuff against teams they lost for example, its silly but possible. Schedule teams to bad times post-elimination is another example).

Reviewer 2: Nothing stops them from not playing at all and entering staff in another tournament and rig it there as well; since hardly anyone ever plays only one tourney in their life.  
I do not think removing this limitation dramatically increases the chances of such bad behavior to occur.

I believe we can trust tournament hosts with this - even more since we now have a proper protocol for players involved to report such malicious behavior with the mentioned form.

Reviewer 4: In my opinion, removing this limitation would change absolutely nothing.
I've seen it being broken in multiple tournaments before, and they've all worked just fine.
Feels just like a worse way of getting less staff members at this point. (Gamemodes other than standard, where tournament players are also active staff members can be a good example)
Especially with the reporting form at this point.
If there IS an actual issue with that, which I heavily doubt, then people can report it.
If there is an overwhelming amount of reports that shows this was a good idea in the first place, sure, let's put it back then.
But for now, it just feels like a limitation to staff members.

And also, you've brought up referees with this: "To join as map selectors or refs I don't like the idea as much because, as former players, they could affect the outcome of the tournament (by picking stuff against teams they lost for example, its silly but possible. Schedule teams to bad times post-elimination is another example)."

But have referees had this much power that they could rig an entire match by themselves?
I have geniunely not seen that in years.
And your first point applies to map selectors, which I don't think most map selectors nowadays do but okay, fair point.
Scheduling teams to bad times however?
In what kind of tournament are referees scheduling matches even?
I have geniunely never understood why players cannot be referees, since, if they are referees: - They don't ref their own match
- If they do blatant rigging, players are just gonna report it to the host, and the host will just overwrite their decision if it's obviously wrong.

Reviewer 2: Updates to the article have been pushed, but this has been left completely unaddressed.

Is there any update? Especially as many tournaments, including osu! World Cups, go against that rule with playtesters?

Reviewer: another bump cause im also interested in what the take is with players that sign up but arent guaranteed to be on a team (for example world cups)

Please read the criteria laid out below, then send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:

- The name of your tournament
- A brief description of your tournament, including its scoring type and any other appropriate features
- Any applicable dates for your tournament
- Any links to public off-site chatrooms or servers used for the event (Discord, etc.)
- A link to the forum thread located under the [Tournaments subforum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament
- Any links to previous iterations of the same tournament (only where applicable)
- A list of the players involved in running the tournament
- A list of players who have registered for the tournament

Please allow **at least a week** for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.

### Criteria

Community-run tournaments which abide by the following hard criteria are eligible for support:

> - The tournament must have at least 64 teams/players minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.

Reviewer 5: I understand one new requirement which is a minimum of 64 players, however 64 teams? If I were to host a single-elimination tournament that has over 200 players signing up but only acquiring 40 teams, wouldn't that make the tournament ineligible for official support? I believe the requirement of 64 teams either isn't clear enough or severely punishes single-elimination tournaments.

Reviewer 6: Should be an oversight, will be corrected later.

Reviewer 2: Also in case you missed it, even without fixing that oversight, your tournament can still have 16 players or teams, as long as it's double-elimination.

Reviewer 3: You can still have less than 64 teams and use a Ro16 double elimination. The tournament would still qualify in that case. I don't think this was an oversight.

Reviewer 6: The thing is the **OR** there, because if the tournament isnt double elimination (like single elimination or swiss-system for example) it is forced to have 64 teams.

Reviewer 3: And that's perfectly fine! Having 64 players in teams of lets say 8 each means you have 8 teams. Single elimination of 8 teams should not be badged imo.

Reviewer 6: I think it goes more on the wording and the concern of @megu-osu points, if we go by number of players (that is totally fine) how this converts into teams? 16 teams of 4 seems like a bare minimum, but for example if we have something similar to test tournament, we would have 4 teams of 16.

The solution should be having this clear on the "teams" side, something like:
```suggestion
- The tournament must have at least 64 players/32 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
```

Also reminding that this would affect other types of tournament systems, like swiss.

- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
    - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappooler or anything similar until that instance of the tournament concludes. You either play or help run, not both.
- The tournament must be run only four or fewer times per year.
- The tournament must have all players screened by the osu! Support Team **before play commences**.
- The tournament must not be "self-reffed".

>     - A dedicated referee must be present during every match.

Author: it infers that a 'dedicated' person (aka not one of the players participating) must be present during every match. they cannot concurrently be both referee and player in the same tournament

>     - A dedicated referee must be present during every match.

Reviewer 7: but what does "be present" mean here? Does the referee need to be in the match lobby itself, or would it suffice that the referee is reachable through bancho chat/discord/other instant messaging system?

- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](LINK GOES HERE) as the very last content of the original post.
- The tournament must not prevent players who pass screening from participation without both ample evidence presented publicly against them, and the approval of the osu! Support Team. This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.

>     - This includes preventing users who are perceived to be 'sandbagging' from play. If this is an issue which you believe affects your tournament, consider altering your sign-up criteria or rank range.

Reviewer 7: I don't quite understand what this section is aiming for. If a tournament listed "not a sandbagger" (whatever that means) in their player sign-up criteria, would they be then abiding by this section of the official support criteria?

And what about invitational tournaments? Are those still eligible for official support like they have been so far, despite seeminlgy being in direct violation of this new section? Or is the invitational fromat considered just another, very strict form of sing-up criteria?

Reviewer 4: Apologies for not answering to your question directly, but I wanted to ask, and since it's about the same line.
In regards to sandbaggers as a whole. Does this mean hosts, or a team of independent players, are entirely denied the possibility to deny sandbaggers from their tournament, even if it's before the registration phase ends, or do we have to send a list of players with proper proof they are sandbagging in the mode they're playing for to osu!support?
This part feels really unclear to me.

Reviewer: > If a tournament listed "not a sandbagger" (whatever that means) in their player sign-up criteria, would they be then abiding by this section of the official support criteria?

unless you somehow get a committee that can come to a concrete conclusion of what makes someone a sandbagger/deranker, these tournaments are way too host biased to be allowed a badge imo

> And what about invitational tournaments? Are those still eligible for official support like they have been so far, despite seeminlgy being in direct violation of this new section? Or is the invitational fromat considered just another, very strict form of sing-up criteria?

i can see the latter being an issue but assuming you're referring to RMOT, an invitational like that should be completely supported like normal

LAN tournaments are also required to add links to VODs and a copy of the bracket of the entire tournament in order to be considered for a badge prize.

## Frequently Asked Questions (FAQ)

### Why does a tournament have to run at least twice or more in order to qualify for profile badges without experienced staff present?

The second iteration of a tournament is often free of a lot of the organizational issues of the first tournament, and has established a solid workflow and staff involved in its production. They are far less likely to peter out or be subject to questionable choices. The tournament is also far more identifiable if it runs more than just once.

For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.

### What is considered 'staff' as far as tournament management goes?

We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.

> Commentators are not considered by us as staff, and may play or interact in tournaments freely so long as they are not directly involved any managerial aspect of running the tournament or violate anything mentioned above.

Reviewer 2: Bit nitpicky, but maybe mention "streamers" too?

Reviewer 6: Streamers dont affect the outcome of a match, as refs and commentators could do.

Reviewer 7: How would a commentator be able to affect the outcome of a match where a streamer would not?

Reviewer 4: Isn't PooN just saying to mention streamers in the line about commentators?

Reviewer 2: > Streamers dont affect the outcome of a match, as refs and commentators could do.

I don't think streamers and commentators affect the outcome of a match.

To clarify:  
In commonly-structured tournaments, streamers are another position players of a badged tournament can apply for. Thus, I suggest to mention it alongside commentators.

Reviewer 6: Yeah, i should made it clearer too, streamers dont directly intervene on matches, commentators dont have issues to participate but there were instances in the past that commentators hated a certain player and pointed bad things of him on stream, I think this goes more on tournament staff behaviour but depending if the player is listening to the stream that could affect their tournament performance.
Back to the point, no issues on making it clear that commentators and streamers shouldnt have issues on playing.

We have historically had issues where unscrupulous organizers have geared tournaments specifically (ie: picked maps they're familiar with, adjusted seeds/brackets, etc) to farm profile badges for them and their friends. While we understand that most groups will not do this, we must enforce this restriction to keep things fair for everyone.

### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems?

Generally no, unless the role they are assuming is not one that could conceivably alter the outcome of the tournament.

A player may conclude play in your tournament and join as a commentator, for example, but not as a referee, organizer, or anything further.

### Why can't badges be awarded to recurring monthly (or weekly) tournaments?

Flooding the game with profile badges dilutes the prestige of the prize for everybody else. We do not want profile badges to be something that people 'farm' for from a few sets of regular, recurring tournaments.

### Can I request a badge for a beatmap contest or other type of competition?

> For beatmap contests, yes, just follow the same rules as above. We highly recommend using Featured Artists for such contests and will flagrantly give preferential treatment to those who do.

Reviewer 8: ```suggestion
For beatmap contests, yes, just follow the same rules as above. We highly recommend using [Featured Artists](https://osu.ppy.sh/beatmaps/artists) for such contests and will flagrantly give preferential treatment to those who do.
```

For other contests, email us and ask personally. We'll try to work something out.

### Can a team-based tournament match be played if a team is missing a player?

> Yes, a match can still be played even if one of the teams is missing one (or more) players.

Reviewer 8: ```suggestion
Yes, a match can still be played even if one of the teams is missing one or more players.
```
brackets are kind of unnecessary here

### Can I request to advertise my tournament on the main menu?

Yes, you can. It's at the osu!team's discretion to decide if your tournament can be advertised in-game, as there are many factors to consider.

The minimum requirements to request in-game advertising are as follows:

> - Your tournament meets the Criteria for Applicable Tournaments that can be found at the top of this article

Reviewer 8: ```suggestion
- Your tournament meets the [Criteria for Applicable Tournaments](#criteria) that can be found at the top of this article
```

# Official tournament support

> The [osu!team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.

Reviewer 9: ```suggestion
The [osu!team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments, which abide by a certain set of criteria and expectations, with extra support, at their discretion.
```
may read better this way i think?

Author: forgot to comment on this but i think your change reads worse, the pauses (commas) sound unnatural to me when read aloud

## Benefits

Officially supported tournaments have access to:

- **Profile badge prizes for first-place winners**
- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [FAQ](#frequently-asked-questions-faq) section below)
- A listing in the [automatic tournament registrations system](https://osu.ppy.sh/community/tournaments)
- A dedicated screening process to prevent players with serious integrity infringements (cheating, multiaccounting, tournament bans) from playing

## Expectations

Community-run tournaments that are receiving this support are expected to abide by the following general ideals:

- Should remain free and fair
- Be as transparent as possible
- Preserve competitive integrity
- Encourage good sportsmanship

## Tournament screening

> A major part of being an officially supported tournament is access to the screening process. Similar to the 'security checks' undertaken by all World Cup players, screening helps prevent people with recent serious infringements or tournament bans from disrupting play.

Reviewer 9: ```suggestion
A major part of being an officially supported tournament is access to the screening process. Similar to the "security checks" undertaken by all World Cup players, screening helps prevent people with recent serious infringements or tournament bans from disrupting play.
```

- Any links to previous iterations of the same tournament (only where applicable)
- A list of the players involved in running the tournament
- A list of players who have registered for the tournament

Please allow **at least a week** for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.

### Criteria

Community-run tournaments which abide by the following hard criteria are eligible for support:

- The tournament must have at least 64 players or 16 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.

>     - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappooler or anything similar until that instance of the tournament concludes. You either play or help run, not both.

Reviewer 9: ```suggestion
    - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappool selector or anything similar until that instance of the tournament concludes. You either play or help run, not both.
```

- The tournament must be run only four or fewer times per year.
- The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
- The tournament must not be "self-reffed".
    - A dedicated referee must be present during every match.
- The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
    - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.

>     - This includes preventing users who are perceived to be 'sandbagging' from play.

Reviewer 9: ```suggestion
    - This includes preventing users who are perceived to be "sandbagging" from play.
```

- All multiplayer games relevant to the tournament must be created with the `!mp make` command, so that they do not expire.
- All matches undertaken for the tournament must be recorded and their history made publicly available on the original forum post for the tournament in a clearly marked link in normal size font or larger.
- In the event that a player becomes restricted during the course of the tournament, you may need to replay certain matches where they were involved in order to preserve the competitive integrity of the tournament.
    - Organizers should contact the account support team to receive information and guidance on these (hopefully rare) cases.

### After the tournament

Once the tournament has concluded, the organizers will need to submit the following to the account support team:

- A list of applicable badge recipients (aka: the winners of your tournament)
- A link to the publicly viewable catalogue of your tournament's match history

LAN tournaments are also required to add links to VODs and a copy of the bracket of the entire tournament in order to be considered for a badge prize.

> ## Frequently Asked Questions (FAQ)

Reviewer 9: ```suggestion
## Frequently asked questions
```
should be okay?

### Why does a tournament have to run at least twice or more in order to qualify for profile badges without experienced staff present?

The second iteration of a tournament is often free of a lot of the organizational issues of the first tournament, and has established a solid workflow and staff involved in its production. They are far less likely to peter out or be subject to questionable choices. The tournament is also far more identifiable if it runs more than just once.

For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.

> ### What is considered 'staff' as far as tournament management goes?

Reviewer 9: ```suggestion
### What is considered "staff" as far as tournament management goes?
```

**Allowing users that have failed screening to play in your tournament will result in an immediate withdrawal of support and will result in future requests for support to be denied.**

## Designing your profile badge

Profile badge images must adhere to the following standards:

- The badge image file MUST be a PNG.
- The badge image MUST be **172x80**px in dimension.
- The badge MUST clearly display the logo, motif or name of your tournament.
- The badge MUST be cleanly designed and of reasonable visual quality.

Please consult the following examples of approved badges:

![](https://assets.ppy.sh/profile-badges/flc2-2020@2x.png)
![](https://assets.ppy.sh/profile-badges/irt2-2020@2x.png)

Badge images that do not meet these standards will be refused with given reasoning, and you may be asked to completely redesign your badge in some cases. This is fairly rare, but should be accounted for.

## Designing your main menu banner

In rare circumstances, some long-running community tournaments may qualify for ingame main-menu banner support. Such images must adhere to the following standards:

- The banner image MUST be a PNG.
- The banner image MUST be **1500x256**px in dimension.

>     - All banner images will be treated as ``2x``, meaning that they will be exported at 50% scale by us in a ``1x`` image.
>     - Design your images appropriately to ensure things are legible at half size.

Reviewer 9: ```suggestion
  - All banner images will be treated as ``2x``, meaning that they will be exported at 50% scale by us in a ``1x`` image.
  - Design your images appropriately to ensure things are legible at half size.
```

Please read the criteria laid out below, then send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:

- The name of your tournament
- A brief description of your tournament, including its scoring type and any other appropriate features
- Any applicable dates for your tournament
- Any links to public off-site chatrooms or servers used for the event (Discord, etc.)
- A link to the forum thread located under the [Tournaments subforum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament
- Any links to previous iterations of the same tournament (only where applicable)
- A list of the players involved in running the tournament
- A list of players who have registered for the tournament

Please allow **at least a week** for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.

### Criteria

Community-run tournaments which abide by the following hard criteria are eligible for support:

> - The tournament must have at least 64 players or 16 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
> - The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
> - Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
>     - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappool selector or anything similar until that instance of the tournament concludes. You either play or help run, not both.
> - The tournament must be run only four or fewer times per year.
> - The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
> - The tournament must not be "self-reffed". A dedicated referee must be present during every match.
> - The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
>     - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
>     - This includes preventing users who are perceived to be "sandbagging" from play.
>         - Should an organizer have valid concerns about the presence of such players affecting the competitive integrity of their tournament, they may raise the issue to the Tournament Committee for a case-by-case review using the link listed below.
> - The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
>     - ```https://pif.ephemeral.ink/tournament-reports```

Reviewer 9: ```suggestion
- The tournament must have at least 64 players or 16 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
  - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappool selector or anything similar until that instance of the tournament concludes. You either play or help run, not both.
- The tournament must be run only four or fewer times per year.
- The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
- The tournament must not be "self-reffed". A dedicated referee must be present during every match.
- The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
  - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
  - This includes preventing users who are perceived to be "sandbagging" from play.
    - Should an organizer have valid concerns about the presence of such players affecting the competitive integrity of their tournament, they may raise the issue to the Tournament Committee for a case-by-case review using the link listed below.
- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
  - ```https://pif.ephemeral.ink/tournament-reports```
```

### During the tournament

If a tournament satisfies the criteria outlined above, it must also abide by the following practices during play:

- Tournament organizers will be expected to monitor the matches of their tournament for toxic behavior or anything that goes against the expectations laid out above, and report it to the account support team.
- Tournament organizers who spot potentially suspicious play during their tournament are **single-handedly responsible for reporting such incidents to the account support team** via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
- All multiplayer games relevant to the tournament must be created with the `!mp make` command, so that they do not expire.
- All matches undertaken for the tournament must be recorded and their history made publicly available on the original forum post for the tournament in a clearly marked link in normal size font or larger.
- In the event that a player becomes restricted during the course of the tournament, you may need to replay certain matches where they were involved in order to preserve the competitive integrity of the tournament.

>     - Organizers should contact the account support team to receive information and guidance on these (hopefully rare) cases.

Reviewer 9: ```suggestion
  - Organizers should contact the account support team to receive information and guidance on these (hopefully rare) cases.
```

# Official tournament support

The [osu!team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.

## Benefits

Officially supported tournaments have access to:

- **Profile badge prizes for first-place winners**
- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [FAQ](#frequently-asked-questions-faq) section below)
- A listing in the [automatic tournament registrations system](https://osu.ppy.sh/community/tournaments)
- A dedicated screening process to prevent players with serious integrity infringements (cheating, multiaccounting, tournament bans) from playing

## Expectations

Community-run tournaments that are receiving this support are expected to abide by the following general ideals:

- Should remain free and fair
- Be as transparent as possible
- Preserve competitive integrity
- Encourage good sportsmanship

> In addition, all promotional material or any services associated with a tournament receiving official support should adhere to the [osu! Community Rules](/wiki/Rules). This includes things like Twitter accounts, Discord servers, and so on.

Reviewer 9: ```suggestion
In addition, all promotional material or any services associated with a tournament receiving official support should adhere to the [osu! community rules](/wiki/Rules). This includes things like Twitter accounts, Discord servers, and so on.
```
i dont think we need to capitalize this

**Allowing users that have failed screening to play in your tournament will result in an immediate withdrawal of support and will result in future requests for support to be denied.**

## Designing your profile badge

Profile badge images must adhere to the following standards:

- The badge image file MUST be a PNG.
- The badge image MUST be **172x80**px in dimension.
- The badge MUST clearly display the logo, motif or name of your tournament.
- The badge MUST be cleanly designed and of reasonable visual quality.

Please consult the following examples of approved badges:

![](https://assets.ppy.sh/profile-badges/flc2-2020@2x.png)
![](https://assets.ppy.sh/profile-badges/irt2-2020@2x.png)

Badge images that do not meet these standards will be refused with given reasoning, and you may be asked to completely redesign your badge in some cases. This is fairly rare, but should be accounted for.

## Designing your main menu banner

In rare circumstances, some long-running community tournaments may qualify for ingame main-menu banner support. Such images must adhere to the following standards:

- The banner image MUST be a PNG.
- The banner image MUST be **1500x256**px in dimension.

>   - All banner images will be treated as ``2x``, meaning that they will be exported at 50% scale by us in a ``1x`` image.

Reviewer 9: ```suggestion
  - All banner images will be treated as `2x`, meaning that they will be exported at 50% scale by us in a `1x` image.
```

- The banner MUST be cleanly designed with an emphasis on being mostly transparent and relatively unobtrusive visually.

Please do not submit a main menu banner with your original request - you will be asked and pointed to this section if you are authorized to make use of one.

## Requesting support

To start with, the request for support **must be made well before the tournament actually commences play.** 

There are no specific timelines for this, but we recommend at least two to three weeks before play commences to account for delays in the screening process. 

You may be required to postpone the start of your tournament during periods of high delay, but the osu!team will make efforts to ensure this does not happen.

Please read the criteria laid out below, then send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:

- The name of your tournament
- A brief description of your tournament, including its scoring type and any other appropriate features
- Any applicable dates for your tournament
- Any links to public off-site chatrooms or servers used for the event (Discord, etc.)

> - A link to the forum thread located under the [Tournaments subforum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament

Reviewer 9: ```suggestion
- A link to the forum thread located under the [Tournaments forum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament
```

- The tournament must be run only four or fewer times per year.
- The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
- The tournament must not be "self-reffed". A dedicated referee must be present during every match.
- The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
  - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
  - This includes preventing users who are perceived to be "sandbagging" from play.
    - Should an organizer have valid concerns about the presence of such players affecting the competitive integrity of their tournament, they may raise the issue to the Tournament Committee for a case-by-case review using the link listed below.
- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.

>   - ```https://pif.ephemeral.ink/tournament-reports```

Reviewer 9: ```suggestion
  - `https://pif.ephemeral.ink/tournament-reports`
```

### Why can't badges be awarded to recurring monthly (or weekly) tournaments?

Flooding the game with profile badges dilutes the prestige of the prize for everybody else. We do not want profile badges to be something that people 'farm' for from a few sets of regular, recurring tournaments.

### Can I request a badge for a beatmap contest or other type of competition?

For beatmap contests, yes, just follow the same rules as above. We highly recommend using [Featured Artists](https://osu.ppy.sh/beatmaps/artists) for such contests and will flagrantly give preferential treatment to those who do.

For other contests, email us and ask personally. We'll try to work something out.

### Can a team-based tournament match be played if a team is missing a player?

Yes, a match can still be played even if one of the teams is missing one or more players.

### Can I request to advertise my tournament on the main menu?

Yes, you can. It's at the osu!team's discretion to decide if your tournament can be advertised in-game, as there are many factors to consider.

The minimum requirements to request in-game advertising are as follows:

> - Your tournament meets the [Criteria for Applicable Tournaments](#criteria) that can be found at the top of this article

Reviewer 9: ```suggestion
- Your tournament meets the [criteria for applicable tournaments](#criteria) that can be found at the top of this article
```

- Tournament organizers who spot potentially suspicious play during their tournament are **single-handedly responsible for reporting such incidents to the account support team** via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
- All multiplayer games relevant to the tournament must be created with the `!mp make` command, so that they do not expire.
- All matches undertaken for the tournament must be recorded and their history made publicly available on the original forum post for the tournament in a clearly marked link in normal size font or larger.
- In the event that a player becomes restricted during the course of the tournament, you may need to replay certain matches where they were involved in order to preserve the competitive integrity of the tournament.
  - Organizers should contact the account support team to receive information and guidance on these (hopefully rare) cases.

### After the tournament

Once the tournament has concluded, the organizers will need to submit the following to the account support team:

> - A list of applicable badge recipients (aka: the winners of your tournament)

Reviewer 10: ```suggestion
- A list of applicable badge recipients (a.k.a. the winners of your tournament)
```

- A link to the publicly viewable catalogue of your tournament's match history

LAN tournaments are also required to add links to VODs and a copy of the bracket of the entire tournament in order to be considered for a badge prize.

## Frequently asked questions

### Why does a tournament have to run at least twice or more in order to qualify for profile badges without experienced staff present?

The second iteration of a tournament is often free of a lot of the organizational issues of the first tournament, and has established a solid workflow and staff involved in its production. They are far less likely to peter out or be subject to questionable choices. The tournament is also far more identifiable if it runs more than just once.

> For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.

Reviewer 10: ```suggestion
For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizeable majority of the team is experienced and accomplished at running tournaments in the past.
```

### What is considered "staff" as far as tournament management goes?

> We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.

Reviewer 10: ```suggestion
We consider referees, mappool selectors, judges, team organisers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
```

Commentators and streamers are not considered by us as staff, and may play or interact in tournaments freely so long as they are not directly involved in any managerial aspect of running the tournament or violate anything mentioned above.

> We have historically had issues where unscrupulous organizers have geared tournaments specifically (ie: picked maps they're familiar with, adjusted seeds/brackets, etc.) to farm profile badges for themselves and their friends. While we understand that most groups will not do this, we must enforce this restriction to keep things fair for everyone.

Reviewer 10: ```suggestion
We have historically had issues where unscrupulous organisers have geared tournaments specifically (i.e. picked maps they're familiar with, adjusted seeds/brackets, etc.) to farm profile badges for themselves and their friends. While we understand that most groups will not do this, we must enforce this restriction to keep things fair for everyone.
```

### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems?

Generally no, unless the role they are assuming is not one that could conceivably alter the outcome of the tournament.

> One main exception is for playtesters collaborating with mappoolers for feedback and replays - this is tentatively allowed, but must be undertaken with care to ensure that it does not skew the tournament's outcome in any unfair or unusual way. We permit this on an expectation of trust and hope that any misuse of this exception would be reported to the tournament committee for prompt review.

Reviewer 10: ```suggestion
One main exception is for playtesters collaborating with mappoolers for feedback and replays—this is tentatively allowed, but must be undertaken with care to ensure that it does not skew the tournament's outcome in any unfair or unusual way. We permit this on an expectation of trust and hope that any misuse of this exception would be reported to the tournament committee for prompt review.
```

> A player may conclude play in your tournament and join as a commentator, for example, but not as a referee, organizer, or anything further.

Reviewer 10: ```suggestion
A player may conclude play in your tournament and join as a commentator, for example, but not as a referee, organiser, or anything further.
```

### Why can't badges be awarded to recurring monthly (or weekly) tournaments?

> Flooding the game with profile badges dilutes the prestige of the prize for everybody else. We do not want profile badges to be something that people 'farm' for from a few sets of regular, recurring tournaments.

Reviewer 10: ```suggestion
Flooding the game with profile badges dilutes the prestige of the prize for everybody else. We do not want profile badges to be something that people 'farm' from a few sets of regular, recurring tournaments.
```
